code/wdb/writedown.q

```diff
@@ -0,0 +1,14 @@
+/-endofperiod function to savedown by integer
+endofperiod:{[currp;nextp;data]
+	.lg.o[`endofperiod;"Received endofperiod. currentperiod, nextperiod and data are ",(string currp),", ", (string nextp),", ", .Q.s1 data]
+        / Need to download sym file to scratch directory if this is Finspace application
+        .lg.o[`createchangeset;"downloading sym file to scratch directory for ",.finspace.database];
+	.aws.get_latest_sym_file[.finspace.database;1_string .wdb.savedir];
+        /- change to integer partition
+	.wdb.currentpartition:(`long$currp);
+	/-call function to savedown tables
```

Comment:
Lets try not to have comments on WHAT the code is doing (unless it's very complicated). Try to stick to commenting on the WHY (if it isn't obvious from the code)